sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/UnsafeRow.java

```diff
@@ -239,16 +239,11 @@ public void setFloat(int ordinal, float value) {
     PlatformDependent.UNSAFE.putFloat(baseObject, getFieldOffset(ordinal), value);
   }
 
-  @Override
-  public int size() {
-    return numFields;
-  }
-
   /**
    * Returns the object for column `i`, which should not be primitive type.
    */
   @Override
-  public Object get(int i) {
+  public Object apply(int i) {
```

Comment:
Actually I'd like to making `apply()` call to `get()` in Row, because it's weird to use `apply()` in Java, and we will have more performance critical code written in Java.